src/utils/normalize-repo-url.ts

```diff
@@ -0,0 +1,24 @@
+import parseGitHubUrl from 'parse-github-url'
```

Comment:
To start with this only supports GitHub, but I believe I've implemented in such a way that only this util would need to be updated to add support for GitLab, etc.